docusaurus/docs/distributing-your-plugin.md

```diff
@@ -3,19 +3,19 @@ id: distributing-your-plugin
 title: Distributing Your Plugin
 ---
 
-When distributing a Grafana plugin either within the community or privately the plugin must be signed so the Grafana application can verify its authenticity. This can be done with the `@grafana/sign-plugin` package.
+To distribute a Grafana plugin either to the community or privately the plugin must be signed so the Grafana application can verify its authenticity. This can be done with the `@grafana/sign-plugin` package.
 
 :::info
 
-It's not necessary to sign a plugin during development. The docker development environment that is scaffolded with `@grafana/create-plugin` caters for running the plugin without a signature.
+It's not necessary to sign a plugin during development. The docker development environment that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature.
 
 :::
 
 ## Initial steps
 
 Before signing a plugin please read the Grafana [plugin publishing and signing criteria](https://grafana.com/docs/grafana/latest/developers/plugins/publishing-and-signing-criteria/) documentation carefully.
```

Comment:
Change "Before signing a plugin please read..." to "Before signing a plugin, read..."